packages/backend/src/app.ts

```diff
@@ -244,15 +249,22 @@ export class App {
       loaders: [new GraphQLFileLoader()]
     })
 
+    const redis = await this.container.use('redis')
+
     // Add resolvers to the schema
-    const schemaWithResolvers = addResolversToSchema({
-      schema,
-      resolvers
-    })
+    const schemaWithMiddleware = applyMiddleware(
+      addResolversToSchema({
+        schema,
+        resolvers
+      }),
+      idempotencyGraphQLMiddleware(
+        createRedisDataStore(redis, 1000 * 60 * 60 * 24)
```

Comment:
ttl for any idempotency key = 24 hrs

Reply:
Can we make that configurable, please? Default can be 24hrs.